pkg/apis/creator/v1/types.go

```diff
@@ -59,6 +59,7 @@ func (ss *ServiceSpec) EmailAddress() string {
 // +k8s:deepcopy-gen=true
 type ServiceMetadata struct {
 	PagerDuty *PagerDutyMetadata `json:"pagerDuty,omitempty"`
+	OpsGenie  *OpsGenieMetadata  `json:"opsGenie,omitempty"`
```

Comment:
This change seems valid, we should just set the team name based on the "team" service attribute rather than misc field. Also, this part should be read-only, we never edit team, editing is done via Microscope UI (at least that's our agreement for now)

Reply:
I don't know what you mean, lets talk IRL.